sql/moz-fx-data-marketing-prod/ga_derived/www_site_hits_v2/query.sql

```diff
@@ -0,0 +1,294 @@
+-- Query for ga_derived.www_site_hits_v2
+            -- For more information on writing queries see:
+            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+WITH get_session_start_time AS (
+  SELECT
+    PARSE_DATE('%Y%m%d', a.event_date) AS date,
+    a.user_pseudo_id || '-' || CAST(
+      (
+        SELECT
+          `value`
+        FROM
+          UNNEST(event_params)
+        WHERE
+          key = 'ga_session_id'
+        LIMIT
+          1
+      ).int_value AS string
+    ) AS visit_identifier,
+    a.user_pseudo_id AS full_visitor_id,
+    MIN(event_timestamp) AS visit_start_time
+  FROM
+    `moz-fx-data-marketing-prod.analytics_313696158.events_*` a
+  WHERE
+    _TABLE_SUFFIX = FORMAT_DATE('%Y%m%d', @submission_date)
+  GROUP BY
+    1,
+    2,
+    3
+),
+get_all_events_in_each_session_staging AS (
+  SELECT
+    PARSE_DATE('%Y%m%d', a.event_date) AS date,
+    a.user_pseudo_id || '-' || CAST(
+      (
+        SELECT
+          `value`
+        FROM
+          UNNEST(event_params)
+        WHERE
+          key = 'ga_session_id'
+        LIMIT
+          1
+      ).int_value AS string
+    ) AS visit_identifier,
+    a.user_pseudo_id AS full_visitor_id,
+    a.event_name,
+    a.event_timestamp,
+    device.category AS device_category,
+    device.operating_system AS operating_system,
+    device.language AS language,
+    device.web_info.browser AS browser,
```

Comment:
Looking at the data, this would be the Mozilla's browser right?

```suggestion
    device.web_info.browser AS app_name,
```

Reply:
This is the browser the user used when they visited Mozilla.org, so it could be like Safari, etc - I have to match the column names to the V1 table unfortunately :(